src/redux/sagas/siteSaga.js

```diff
@@ -1,18 +1,43 @@
-import { put, takeLatest, takeEvery } from 'redux-saga/effects';
+import { takeEvery, put as dispatch } from 'redux-saga/effects';
```

Comment:
If you're going to call 'put' 'dispatch' then can you actually refer to it as put? Have you tried running this code?

Reply:
Just use 'put', since that's what it's called in sagas.